src/structures/GuildChannel.js

```diff
@@ -213,7 +213,7 @@ class GuildChannel extends Channel {
   }
 
   /**
-   * Updates Overwrites for a user or role in this channel. (creates if non-existent)
+   * Updates permission overwrites for a user or role in this channel, or creates an entry if not already present
```

Comment:
```suggestion
   * Updates permission overwrites for a user or role in this channel, or creates an entry if not already present.
```
full stop should be at the end here right, since its a method description